### PR TITLE
Update customized codes

### PR DESCRIPTION
**Pull request by yifan-zhou922**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/purview/Azure.Analytics.Purview.DataMap/src/Custom/ImportBusinessMetadataRequest.Serialization.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    // Implement custom serialization code to compose a request with Content-Type:
+    // multipart/form-data, which currently cannot be auto-generated.
+
+    internal virtual RequestContent ToRequestContent()
+    {
+        MultipartFormDataContent content = new MutipartFormDataContent();
+        content.Add(MultipartContent.Create(File), new Dictionary<string, string>()
+        {
+            ["Content-Disposition"] = $"form-data; name=file; filename=file",
+            ["Content-Type"] = "multipart/form-data",
```
Comment (chunyu3) [inline sdk/purview/Azure.Analytics.Purview.DataMap/src/Custom/ImportBusinessMetadataRequest.Serialization.cs:22]: The `content-type` should be `application/octet-stream`. This is to define the part for `File`, and the content-type is the type of the File data

File: sdk/purview/Azure.Analytics.Purview.DataMap/src/Custom/ImportBusinessMetadataRequest.Serialization.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        });
```
Comment (chunyu3) [inline sdk/purview/Azure.Analytics.Purview.DataMap/src/Custom/ImportBusinessMetadataRequest.Serialization.cs:23]:
```suggestion
        content.Add(MultipartContent.Create(File), "file", "file", null);
```

File: sdk/purview/Azure.Analytics.Purview.DataMap/src/Custom/Entity.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        request.Method = RequestMethod.Post;
+        var uri = new RawRequestUriBuilder();
+        uri.Reset(_endpoint);
+        uri.AppendRaw("/datamap/api", false);
+        uri.AppendPath("/atlas/v2/entity/businessmetadata/import", false);
+        uri.AppendQuery("api-version", _apiVersion, true);
+        request.Uri = uri;
+        request.Headers.Add("Accept", "multipart/form-data");
+        request.Headers.Add("content-type", "multipart/form-data");
+        request.Content = content;
```
Comment (chunyu3) [inline sdk/purview/Azure.Analytics.Purview.DataMap/src/Custom/Entity.cs:35]:
```suggestion
        request.Content = content;
        (content as MultipartFormDataContent).ApplyToRequest(request);
```

File: sdk/purview/Azure.Analytics.Purview.DataMap/src/Custom/ImportBusinessMetadataRequest.Serialization.cs (hunk 2 above)
Comment (yifan-zhou922) [reply sdk/purview/Azure.Analytics.Purview.DataMap/src/Custom/ImportBusinessMetadataRequest.Serialization.cs]: updated

**Pull request closed without merging**